js/freewheel.js

```diff
@@ -272,6 +274,10 @@ OO.Ads.manager(function(_, $) {
       fwContext.setParameter(tv.freewheel.SDK.PARAMETER_RENDERER_VIDEO_DISPLAY_CONTROLS_WHEN_PAUSE, false, tv.freewheel.SDK.PARAMETER_LEVEL_GLOBAL);
       fwContext.setParameter(tv.freewheel.SDK.PARAMETER_RENDERER_VIDEO_CLICK_DETECTION, true, tv.freewheel.SDK.PARAMETER_LEVEL_GLOBAL);
 
+      if (bitrateOverride) {
+        fwContext.setParameter(tv.freewheel.SDK.PARAMETER_DESIRED_BITRATE, bitrateOverride, tv.freewheel.SDK.PARAMETER_LEVEL_OVERRIDE);
```

Comment:
Just curious, what happens when you set a bitrate that is not supported by the ad? Does it round to the nearest bitrate?